src/main/java/com/conveyal/gtfs/util/PolylineUtils.java

```diff
@@ -0,0 +1,325 @@
+package com.conveyal.gtfs.util;
+
+import com.sun.istack.internal.NotNull;
```

Comment:
This is causing a compilation error in the [Travis build](https://travis-ci.com/github/conveyal/gtfs-lib/builds/205469968).